wallet/src/repository/wallet/mod.rs

```diff
@@ -1117,6 +1125,12 @@ where
         Ok(address)
     }
 
+    fn _set_stop_syncing(&self, state: &mut State) -> Result<()> {
+        state.stop_syncing = true;
+
+        Ok(())
+    }
+
```

Comment:
If it is not used maybe I would remove this function and move the logic to `set_stop_syncing`. What do you think guys?